perceval/backend.py

```diff
@@ -379,26 +399,28 @@ class BackendCommandArgumentParser:
     types of authentication can be set during the initialization
     of the instance.
 
-    :param categories: set category argument
+    :param backend: backend object
     :param from_date: set from_date argument
     :param to_date: set to_date argument
     :param offset: set offset argument
     :param basic_auth: set basic authentication arguments
     :param token_auth: set token/key authentication arguments
     :param archive: set archiving arguments
     :param aliases: define aliases for parsed arguments
+    :param blacklist: set a list of items IDs to be filtered out
 
-    :raises AttributeArror: when both `from_date` and `offset` are set
+    :raises AttributeError: when both `from_date` and `offset` are set
         to `True`
     """
 
-    def __init__(self, categories, from_date=False, to_date=False, offset=False,
+    def __init__(self, backend, from_date=False, to_date=False, offset=False,
                  basic_auth=False, token_auth=False, archive=False,
-                 aliases=None):
+                 aliases=None, blacklist=False):
         self._from_date = from_date
         self._to_date = to_date
         self._archive = archive
-        self._categories = categories
+        self._categories = backend.CATEGORIES
+        self._origin_unique_field = backend.ORIGIN_UNIQUE_FIELD if backend.ORIGIN_UNIQUE_FIELD else None
```

Comment:
I think this variable is not needed. You should call to the property you defined in the backend instead.

Moreover, which cases these check is going to be `None`? I think to assign `None` it's not necessary.